app/helpers/tags_helper.rb

```diff
@@ -0,0 +1,9 @@
+module TagsHelper
+  def food_link(food_record)
+    if food_record.present?
```

Comment:
下記じゃないですかね？
でないと、food_recordがnilの場合もelseに入るので、`food_record_path(food_record)` でエラーになりそうな気がします。
```suggestion
    if food_record. food_name .present?
```

Reply:
確かに、料理名の存在判定が必要でした。ありがとうございます。
food_recordがnilの場合の話ですが、

app/views/tags/_food.html.erb
```
<% if food_records.present? %>
  <% food_records.each do |food_record|%>
    <%= food_link(food_record) %>
  <% end %>
<% else %>
  登録された料理はありません。
<% end %>
```